build.sbt

```diff
@@ -64,3 +64,7 @@ releaseProcess := Seq[ReleaseStep](
   commitNextVersion,
   pushChanges
 )
+
+PlayKeys.devSettings := Seq(
```

Comment:
Just wondering, but is this really required for the runtime behaviour of the app? I'd think the setting in `application.conf` would be enough.

Reply:
This was also confusing for me. It seems in dev mode certain settings can't be configured via `application.conf`.

https://www.playframework.com/documentation/2.6.x/ConfigFile#Using-with-the-run-command

https://github.com/playframework/playframework/issues/7617

Reply:
Since this is an extreme edge case, I wouldn't bother with it in dev mode. Let's drop this snippet for now.